frame/system/src/lib.rs

```diff
@@ -461,6 +464,10 @@ decl_storage! {
 		/// True if we have upgraded so that `type RefCount` is `u32`. False (default) if not.
 		UpgradedToU32RefCount build(|_| true): bool;
 
+		/// True if we have upgraded so that AccountInfo contains two types of `RefCount`. False
+		/// (default) if not.
+		UpgradedToDualRefCount build(|_| true): bool;
```

Comment:
Note: you can use the pallet storage version now.

Reply:
IMHO best would be to have an isolated function which does this migration, and we can also do a version bump.
the isolated function wouldn't depend on system so that there is no burden to maintain it.

Similarly to https://github.com/paritytech/substrate/pull/7040/files#diff-3acd0cb73093b0cc01acfd0dc5646b31a309e57526f18d67d7e9d8708c325dbb

I understand it is less handy for polkadot so I'm ok to have it as it is now. But from a substrate perspective having an isolated code anyone can pickup is better, than a inline migration which will be removed after polkadot runtime upgrade, or removed when it start being annoying to maintain.

Reply:
Isolated function is done.